bm.js

```diff
@@ -7,7 +7,12 @@ var fs = require('fs');
 var argv = require('minimist')(process.argv.slice(2));
 
 var userhome = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];
-var rc_path = userhome+"/.bmndrrc";
+if (process.env.XDG_CONFIG_HOME) {
+  var rc_path = process.env.XDG_CONFIG_HOME+"/.bmndrrc";
+} else {
+  var rc_path = userhome+"/.bmndrrc";
```

Comment:
As far as I can tell, `userhome` is now only being used here on L13. If so, its scope could be reduced.

Reply:
As in something like this? 
```
if (process.env.XDG_CONFIG_HOME) {
  var rc_path = process.env.XDG_CONFIG_HOME+"/.bmndrrc";
} else {
  var userhome = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];
  var rc_path = userhome+"/.bmndrrc";
}
```